### ISSUE TITLE
Flip panel issue

### ISSUE THREAD

**Issue by G-Guy** [BEEmod/BEE2-items#3585]
Hello. I have an issue with one of default items, the flip panel. When I am in test editor, the panels are set to black side by me. However when I build the test they are white side for some reason. Disabling or enabling start reversed in flip panel options doesn't change anything. I would like to note that this issue appeared ONLY after I started using BEE2.4. It wasn't present without it installed.

Item icon - https://freeimage.host/i/KWlE6g
Editor screenshot - https://freeimage.host/i/KWljup
In-game screenshot - https://freeimage.host/i/KWlwwN

Please do something about it.

**Comment by Konclan** [comment]
Duplicate of #3467
